src/json/icons-map.json

```diff
@@ -27,17 +21,19 @@
         "crit": "\u0063",
         "dalan-bankleft": "\u005b",
         "dalan-bankright": "\u005d",
+        "device": "\u0042",
         "doubleturretarc": "\u0071",
-        "elite": "\u0045",
```

Comment:
Can we keep `elite` as an alias for `talent`?

Reply:
I need to understand why first? It is called talent in 2nd edition. I renamed bomb to device as well btw.

Reply:
Actually, on second thought that's fine.